optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/PartitionSolver.java

```diff
@@ -64,6 +64,11 @@ public Score getBestScore() {
         return solverScope.getBestScore();
     }
 
+    @Override
+    public String explainBestScore() {
+        throw new UnsupportedOperationException();
```

Comment:
Does it mean it doesn't make sense to support this operation or it's just not supported yet? The exception could provide some explanation.